### ISSUE TITLE
[Feature]: Add warning about Stable Horde logging prompts and generations

### ISSUE THREAD

**Issue by shankarsivarajan** [sdwebui-w-horde/sd-webui-stable-horde-worker#107]
### Is there existing issue for this?

- [X] I didn't find similar issue, also checked the latest commit.

### Feature Description

The logo of this project, 
<img src="https://raw.githubusercontent.com/sdwebui-w-horde/sd-webui-stable-horde-worker/master/logo.png" width = 256 align="center">
suggests that it's intended for generating anime images.

I'd suggest adding an explicit warning that, contrary to the intuitions one might have about free and open source programs, Stable Horde is designed to make malicious changes to your inputs and log your prompts, as documented in [this](https://dbzer0.com/blog/ai-powered-anti-csam-filter-for-stable-diffusion/) blog post from the developer, along with the ostensible justification ("think of the children!") specifically noting its use against images in the anime style.

### Proposed Solution

- Add something like `⚠️ Stable Horde logs your prompts and might report to you to the FBI if the developer decides your favored art style makes you a "pedo"`

### Additional Information

See [this](reddit.com/r/PygmalionAI/comments/11rcl4l/psa_stable_horde_has_a_mandatory_fake_csam_filter/jd76wya/) thread, where the developer (u/dbzer0/) calls anyone concerned about privacy "fash"

**Comment by db0** [comment]
I don't "call anyone concerned about privacy fash". I called people praising fascists, fash.

**Comment by shankarsivarajan** [comment]
The questions you refused to answer:

1. Does your anti-"CSAM" filter, as implemented, strictly block and only block photorealistic depictions of only minors that could be indistinguishable from actual photos of sexual abuse with a high degree of accuracy (in particular, minimal false positives, that is minimal implication of innocent people)?

2. What are you doing with all of these logs you generate when people trigger this filter?

**Comment by MaikoTan** [comment]
I don't know what is going on. My English is poor so I can only understand the meaning with machine translation.
Since the user of this extension would be the service provider side, I don't think the warning message you mentioned above would help the end users. Or is there something I missed out?
Instead, if I add the news from stable horde onto the extension page, would it satisfy you?

**Comment by shankarsivarajan** [comment]
Yes, people knowing exactly what they're participating in is all I'm asking. Thank you!

**Comment by db0** [comment]
> Instead, if I add the news from stable horde onto the extension page, would it satisfy you?

There is no news. The way the AI horde worked has never changed. This is someone spreading FUD
